assets/yq-for-nmdc_schema_accepting_legacy_ids.txt

```diff
@@ -1,2 +1,8 @@
-'(.classes.[].slot_usage.[] | select(has("pattern")).pattern) = ".*" '
-'(.classes.[].slot_usage.[] | select(has("structured_pattern")).structured_pattern.syntax) = ".*" '
+# '(.classes.[].slot_usage.[] | select(has("pattern")).pattern) = ".*" '
+# '(.classes.[].slot_usage.[] | select(has("structured_pattern")).structured_pattern.syntax) = ".*" '
```

Comment:
If you are keeping this only so you can easily restore it (and not so you can read it when reading the rest of the code), I'd prefer it be deleted. It will still be available in the Git history.